ui/app/components/dropdowns/components/network-dropdown-icon.js

```diff
@@ -0,0 +1,32 @@
+const inherits = require('util').inherits
+const Component = require('react').Component
+const h = require('react-hyperscript')
+
+
+inherits(NetworkDropdownIcon, Component)
+module.exports = NetworkDropdownIcon
+
+function NetworkDropdownIcon () {
+  Component.call(this)
+}
+
+NetworkDropdownIcon.prototype.render = function () {
+  const {
+    backgroundColor,
+    isSelected,
+    innerBorder,
+    nonSelectBackgroundColor
+  } = this.props
+
+  return h('.menu-icon-circle-select', {
+    style: {
+      border: isSelected && '1px solid white',
+      background: isSelected ? 'rgba(100, 100, 100, 0.4)' : 'none',
+    },
+  }, h('.menu-icon-circle', {
+    style: {
+      background: isSelected ? backgroundColor : nonSelectBackgroundColor,
```

Comment:
it's more intuitive to reason `backgroundColor` and `activeBackgroundColor`. 

Also, can we try and see if we can use change opacity of the non-active color to different selected state? This will allow us to use less props and control styling using a BEM className `menu-icon-circle-select--active` and sass, instead of inline-style.